wbooks/src/app/screens/BookCard/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+import { white } from '@constants/colors';
+
+export default StyleSheet.create({
+  wrapper: {
+    backgroundColor: white,
+    borderRadius: 5,
+    flexDirection: 'row',
+    marginTop: 17,
+    height: 90,
+    width: 335
+  },
+  image: {
+    height: 60,
+    width: 40,
+    marginLeft: 30,
+    marginVertical: 15
+  },
+  textWrapper: {
+    flex: 1,
+    marginLeft: 20,
+    marginTop: 15
+  },
+  title: {
+    fontSize: 17
```

Comment:
and the `fontWeight` prop? If you are using the custom rn font you can use the `fontWeight` prop (OSDE On use Custom fonts)